test/SerialIOTest.cpp

```diff
@@ -722,7 +722,7 @@ void fileBased_write_test(const std::string & backend)
             o.flush();
         }
 
-        o.setOpenPMDextension(1);
+        //o.setOpenPMDextension(1); // this will make iteration 1&2 have extension 0.
```

Comment:
With the root cause found, can we activate this "late" extension setting again, please? The comment behind this was only true for ADIOS1, according to my tests.

Reply:
ok, found a solution.